cli/setup.go

```diff
@@ -885,3 +909,52 @@ func (opts *setupOptionsType) maybeAddHostLookup() {
 			route, err.Error())
 	}
 }
+
+func (opts *setupOptionsType) installDemoCertificateLocalTrust() error {
+	_, err := os.Stat(getMenderDemoCertPath())
+	if err != nil {
+		return errors.Wrapf(err,
+			"Cannot stat file %q", getMenderDemoCertPath())
+	}
+
+	s, err := os.Open(getMenderDemoCertPath())
+	if err != nil {
+		return errors.Wrapf(err,
+			"Cannot open file %q", getMenderDemoCertPath())
+	}
+	defer s.Close()
+
+	dir := filepath.Base(getLocalTrustMenderCertPath())
+	_, err = os.Stat(dir)
+	if os.IsNotExist(err) {
+		err := os.MkdirAll(dir, 0755)
+		if err != nil {
+			return errors.Wrapf(err,
+				"Cannot create directory %q", dir)
+		}
+	}
+
+	d, err := os.OpenFile(getLocalTrustMenderCertPath(), os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0755)
```

Comment:
Why 0755 mode?

Reply:
Changing it to `444` to align it with yocto install. Thanks.